$NetBSD$

These two translations causes problems with msgmerge in 0.16.3; remove them.

--- po/LINGUAS.orig	2012-11-13 13:00:26.000000000 +0000
+++ po/LINGUAS
@@ -11,7 +11,6 @@ de
 el
 en_GB
 eo
-es
 et
 fi
 fr
@@ -29,7 +28,6 @@ lt
 lv
 nb
 nl
-pa
 pl
 pt
 pt_BR
